src/Take.Blip.Builder/Actions/ProcessCommand/ProcessCommandAction.cs

```diff
@@ -30,14 +31,20 @@ public async Task ExecuteAsync(IContext context, JObject settings, CancellationT
             if (settings == null) throw new ArgumentNullException(nameof(settings), $"The settings are required for '{nameof(ProcessCommandAction)}' action");
 
             string variable = null;
+            var ignoreTunnelOwnerContext = false.ToString();
 
             if (settings.TryGetValue(nameof(variable), out var variableToken))
             {
                 variable = variableToken.ToString().Trim('"');
             }
+            if (settings.TryGetValue(Constants.IGNORE_OWNER_CONTEXT, out var ignoreOwnerContextToken))
+            {
+                ignoreTunnelOwnerContext = ignoreOwnerContextToken.ToString();
+            }
 
             var command = ConvertToCommand(settings);
             command.Id = EnvelopeId.NewId();
+            command.Metadata.Add($"builder.{Constants.IGNORE_OWNER_CONTEXT}", ignoreTunnelOwnerContext.ToString());
```

Comment:
You don't need to set a metadata for it, just set the command From to _applicationIdentity

Reply:
Actually, it is already possible to send the command "from" from flow JSON. 

Thanks for the suggestion

Reply:
Lol you are right